grants/araeagles/team.md

```diff
@@ -0,0 +1,27 @@
+## Team
+
+### John Doe — Project Lead
+
+John has 20 years of experience coding Solidity. He bought Bitcoin in 1995, therefore inventing the blockchain. He admits to be the real Craig Wright.
+
+**Commitment**: Full time
+
+**Socials**: [GitHub](#) [Twitter](#)
+
+### Satoshi Nakamoto — Frontend Dev
```

Comment:
Don't know about Satoshi's experience with React. Heard rumors he is more of a JQuery guy...